mwdb/resources/__init__.py

```diff
@@ -99,3 +100,27 @@ def load_schema(request_data, schema):
         raise BadRequest(f"JSONDecodeError: {json_decode_err}")
 
     return obj
+
+
+def get_shares_group(upload_as):
```

Comment:
Let's rename it to something that describes better the purpose and provide more documentation.
```suggestion
def get_shares_for_upload(upload_as):
    """
    Translates 'upload_as' value from API into list of groups that
    object will be shared with
    """
```